src/patternfly/components/Card/card.scss

```
@@ -232,11 +248,18 @@
     --#{$card}--BorderColor: var(--#{$card}--m-secondary--BorderColor);
     --#{$card}--BorderWidth: var(--#{$card}--m-secondary--BorderWidth);
     --#{$card}--BackgroundColor: var(--#{$card}--m-secondary--BackgroundColor);
+
+    &.pf-m-selectable:not(.pf-m-current):not(.pf-m-selected),
```

Stylelint (17.3.0)

[error] Expected complex :not() pseudo-class notation (selector-not-notation)

(selector-not-notation)